src/components/InputEmailChip.jsx

```diff
@@ -81,6 +79,12 @@ export const InputEmailChip = ({
     }
   };
 
+  const handleOnEndEditing = event => {
+    if (event?.nativeEvent?.text) {
+      checkAndUpdateEmails(event?.nativeEvent?.text);
+    }
```

Comment:
I think event will never be undefined.

```suggestion
      checkAndUpdateEmails(event.nativeEvent.text);
```